python/src/test/resources/JdbcToBigQueryYamlIT.yaml

```diff
@@ -0,0 +1,16 @@
+pipeline:
+  type: chain
+  transforms:
+    - type: ReadFromPostgres
+      config:
+        driver_class_name: "org.postgresql.Driver"
```

Comment:
ReadFromPostgres already has driver_class_name defined: https://github.com/apache/beam/blob/feb9cda68638b96f2c8fbfbab9d5408cc4801502/sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcUtil.java#L102

one possibility is that providing driver_class_name manually without providing driver_jars can cause error

Reply:
postgres and mysql drivers jar is shipped with expansion service so no need to provide driver jar for these two Jdbc providers